### PR TITLE
Add 6-digits zero padding for transaction costs

### PR DESCRIPTION
**Pull request by shawnxie999**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After
Before 
image
After
image

## Test Plan
Tested decimal numbers that include and no include decimals

### CODE REVIEW

File: src/containers/shared/utils.js [hunk 1]
```diff
@@ -124,11 +124,12 @@ export const localizeNumber = (num, lang = 'en-US', options = {}) => {
   return new Intl.NumberFormat(lang, config).format(number);
 };
 
-export function formatPrice(number, lang = 'en-US', currency = 'USD', decimals = 4) {
```
Comment (ledhed2222) [inline src/containers/shared/utils.js:127]: why would we ever pad with something other than `0`? if there's no need for it now, having the additional parameter is just noise. Let's get rid of it.

**Review by ledhed2222 — CHANGES_REQUESTED** [review]
see comment

File: src/containers/shared/utils.js (hunk 1 above)
Comment (ckniffen) [reply src/containers/shared/utils.js:127]: It isnt what to pad with but how many places to pad.

File: src/containers/shared/utils.js [hunk 2]
```diff
@@ -58,6 +58,13 @@ const NUMBER_DEFAULT_OPTIONS = {
   useGrouping: true,
 };
 
+const FORMAT_PRICE_OPTIONS = {
+  lang: 'en-US',
+  currency: 'USD',
+  decimals: 4,
+  padding: 0,
+};
```
[the remark this answers is not in the document]
Comment (ledhed2222) [reply src/containers/shared/utils.js:66]: the reason here is that there are not the options for format price, that would be an array:

```javascript
const FORMAT_PRICE_OPTIONS = [
  'lang',
  'currency',
  'decimals',
  'padding',
]
```

these are the defaults for the options.

**Review by ledhed2222 — CHANGES_REQUESTED** [review]

**Review by ledhed2222 — APPROVED** [review]

**Pull request merged**